translations/mod-fqm-manager/en.json

```diff
@@ -717,15 +717,15 @@
   "entityType.simple_item_details.circulation_notes": "Circulation notes",
   "entityType.simple_item_details.circulation_notes.date": "Date",
   "entityType.simple_item_details.circulation_notes.date._qualified": "Circulation note date",
-  "entityType.simple_item_details.circulation_notes.id": "ID",
-  "entityType.simple_item_details.circulation_notes.id._qualified": "Circulation note ID",
+  "entityType.simple_item_details.circulation_notes.id": "Circulation note UUID",
+  "entityType.simple_item_details.circulation_notes.id._qualified": "Circulation note UUID",
   "entityType.simple_item_details.circulation_notes.note": "Note",
   "entityType.simple_item_details.circulation_notes.note._qualified": "Circulation note note",
   "entityType.simple_item_details.circulation_notes.note_type": "Note type",
   "entityType.simple_item_details.circulation_notes.note_type._qualified": "Circulation note note type",
   "entityType.simple_item_details.circulation_notes.source": "Source",
   "entityType.simple_item_details.circulation_notes.source._qualified": "Circulation note source",
-  "entityType.simple_item_details.circulation_notes.source.id": "ID",
+  "entityType.simple_item_details.circulation_notes.source.id": "Source ID",
```

Comment:
Not sure if this ID is actually a UUID or not, but if it is then it should be "Source UUID". Otherwise ignore this!